src/goto-programs/remove_virtual_functions.cpp

```diff
@@ -540,12 +574,14 @@ void remove_virtual_functions(
   rvf(goto_functions);
 }
 
+/// Remove virtual function calls from the specified model
 void remove_virtual_functions(goto_modelt &goto_model)
 {
   remove_virtual_functions(
     goto_model.symbol_table, goto_model.goto_functions);
 }
 
+/// Remove virtual function calls from the specified model
```

Comment:
You can use `\copydoc remove_virtual_functions` rather than actually copy-pasting the docs

Reply:
I think I can't do that because the function differs a bit (it's not a method, takes in more arguments).